plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/ServletClasspathProviderTest.java

```diff
@@ -1,24 +1,45 @@
 package com.google.cloud.tools.eclipse.appengine.localserver;
 
+import static org.mockito.Mockito.when;
+
+import java.nio.file.Paths;
+
 import org.eclipse.core.resources.IProject;
 import org.eclipse.core.runtime.CoreException;
 import org.eclipse.jdt.core.IClasspathEntry;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
 import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
 
-// Must Run as JUnit Plug-in Test in Eclipse.
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdk;
+import com.google.cloud.tools.eclipse.sdk.CloudSdkProvider;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
@elharo @briandealwis PTAL. I made a few more changes so the unit/integration tests pass. After these changes, I think this is ready for submission.